electrumx/server/env.py

```diff
@@ -76,7 +76,8 @@ def __init__(self, coin=None):
         self.blacklist_url = self.default('BLACKLIST_URL', self.coin.BLACKLIST_URL)
         self.cache_MB = self.integer('CACHE_MB', 1200)
         self.reorg_limit = self.integer('REORG_LIMIT', self.coin.REORG_LIMIT)
-
+        self.history_flush_count_max = self.integer('HISTORY_FLUSH_COUNT_MAX', 60000)  # deactivated if > 65535
```

Comment:
What is the point of this env var? Why would one set it to lower than 65535?

Reply:
Assume you want to do an urgent transaction via your private electrumx server but notice it's down. Instead of waiting for hours you can just increase that value and proceed. Of course you will have to compact eventually. I thought that would be a nice option and it is not too much effort. I hope I got the way of passing the environment info right along the rest, so far I have only seen this using shared config modules.